about.html

```diff
@@ -1,94 +1,142 @@
+                    </h1>
+                    <p class="about__head--qoute">
+                        Be strong, and let your heart take courage, all you who wait for the
+                        LORD!
+                    </p>
+                    <p class="about__head--verse">Psalm 31:24</p>
+                </div>
+                <div class="about__desc">
+                    <p class="about__desc--content">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime
+                        mollitia, molestiae quas vel sint commodi repudiandae consequuntur
+                        voluptatum laborum numquam blanditiis harum quisquam eius sed odit
+                        fugiat iusto fuga praesentium optio, eaque rerum! Provident
+                        similique accusantium nemo autem.
+                    </p>
+                </div>
+                <div class="about__contact">
+                    <p class="about__contact--guide">
+                        Please contact us via Email for any further questions about the
+                        convention
+                    </p>
+                    <p class="about__contact--email">abzzzzz@convention.com</p>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="about__logo">
+                <h2 class="about__logo--title">
+                    Faith Apostolic Lighthouse Annual Convention 2021 Logo
+                </h2>
+                <hr class="about__logo--line" />
+                <p class="about__logo--desc">
+                    The logo of FAL Annual Convention was designed by a volenteer from
+                    our Kingston chapter.
+                </p>
+                <img src="images/LogoMakr-7y5JLd.png" alt="faith apostolic logo" class="about__logo--logo" />
+            </div>
+        </section>
+        <hr class="divide">
+        <section>
+            <div class="about__summit">
+                <h2 class="about__summit--title">
+                    See the past FAL Annual Conventions
+                </h2>
+                <hr class="about__summit--line" />
+                <p class="about__summit--desc">
+                    Delve into the past experiences of our last conventions which took
+                    place at the Montego Bay Convention Center.
+                </p>
+                <div class="about__summit--cont">
+                    <div class="about__summit--img1">
+                        <div class="shade">
+                            <p class="year">2020</p>
+                            <p class="name">FALAC Montego Bay Jamaica</p>
+                        </div>
+                    </div>
+                    <div class="about__summit--img2">
+                        <div class="shade">
+                            <p class="year">2019</p>
+                            <p class="name">FALAC Montego Bay Jamaica</p>
+                        </div>
```

Comment:
- [ ] These images should be ordered in the center of the container. You should reduce the gap between them.

The Design Template :arrow_down: 
![Screenshot 2021-07-01 at 23-34-59 CC Global Summit 2015](https://user-images.githubusercontent.com/1642014/124189229-14f7d500-dac9-11eb-9142-eeb06d4b96dc.png)

Your Project :arrow_down: 
![Screenshot 2021-07-01 at 23-34-48 Document](https://user-images.githubusercontent.com/1642014/124189234-16c19880-dac9-11eb-85d4-e543fd0aa4cc.png)